share/mk/src.opts.mk

```diff
@@ -369,9 +369,6 @@ BROKEN_OPTIONS+=NS_CACHING
 .endif
 
 .if ${__T:Mriscv*c*}
-# Compiler crash:
-# Skip until https://github.com/CTSRD-CHERI/llvm-project/issues/379 is fixed.
-BROKEN_OPTIONS+=LIBCPLUSPLUS CXX
```

Comment:
Hmm maybe we should keep GOOGLETEST disabled, those extremely long link/compile are annoying.

Reply:
That’s something for cheribuild to consider, not CheriBSD. Can always just disable tests.

Reply:
Or rather it should be upstream freebsd since I believe it also affects the in-tree toolchain. I guess the build is so slow anyway that no one noticed